test/nodejs/errorreporting/errorreporting.js

```diff
@@ -0,0 +1,255 @@
+        console.log(errorGroupStat.group.name + ' ' +
+            errorGroupStat.group.group_id + ' ' +
+            errorGroupStat.count);
+        groups.push(errorGroupStat);
+      }).on('end', function() {
+        var firstErrorGroupId = groups[0].group.group_id;
+        console.log('firstErrorGroupId = ' + firstErrorGroupId);
+        callback(null, firstErrorGroupId);
+      }).on('error', callback);
+}
+
+function listErrorGroupEvents(errorGroupId, callback) {
+  var api = clouderrorreportingV1beta1.errorStatsServiceApi();
+  var formattedProjectName = api.projectPath(PROJECT_NAME);
+  var timeRange = {
+    period: clouderrorreportingV1beta1.QueryTimeRange.Period.PERIOD_1_HOUR
+  };
+
+  console.log('Listing all the error events for error group ' + errorGroupId);
+  api.listEvents(formattedProjectName, errorGroupId,
+      {timeRange: timeRange}).on('data', function(errorEvent) {
+        console.log(errorEvent.message + ' ' +
+            errorEvent.service_context.service + ' ' +
+            errorEvent.service_context.version);
+        console.log(errorEvent.context.user);
+        console.log(errorEvent.context.http_request.method + ' ' +
+            errorEvent.context.http_request.url);
+        console.log(errorEvent.context.report_location.file_path + ' ' +
+            errorEvent.context.report_location.line_number + ' ' +
+            errorEvent.context.report_location.function_name);
+      }).on('end', function() {
+        callback(null, errorGroupId);
+      }).on('error', callback);
+}
+
+function updateErrorGroup(errorGroupId, callback) {
+  var api = clouderrorreportingV1beta1.errorGroupServiceApi();
+  var formattedGroupName = api.groupPath(PROJECT_NAME, errorGroupId);
+
+  console.log('Updating error group ' + formattedGroupName);
+  var issues = [
+    {
+      url: "https://github.com/testuser/project/issues/update1"
+    }];
+  var errorGroup = {
+    name: formattedGroupName,
+    group_id: errorGroupId,
+    tracking_issues: issues
+  };
+  api.updateGroup(errorGroup, function(err, response) {
+    if (err) {
+      console.error(err);
+    }
+    callback(err, response.group_id);
+  });
+}
+
+/**
+ * Integration test for ErrorGroupServiceApi.getGroup,
+ *                     ErrorGroupServiceApi.updateGroup
+ * @param {String} errorGroupId - error group id
+ */
+function getErrorGroup(errorGroupId, callback) {
+  var api = clouderrorreportingV1beta1.errorGroupServiceApi();
+  var formattedGroupName = api.groupPath(PROJECT_NAME, errorGroupId);
+
+  console.log('Getting error group ' + formattedGroupName);
+  api.getGroup(formattedGroupName, function(err, response) {
+    if (err) {
+      console.error(err);
+      callback(err);
+      return;
+    }
+    console.log(response.name + ' ' + response.group_id);
+    console.log('Associated issues');
+    for (var i = 0; i < response.tracking_issues.length; i++) {
+      console.log(response.tracking_issues[i].url);
+    }
+    callback(null, 'Done');
```

Comment:
In that case `console.log('listErrorInfo Done');` in line 134 and removing this would be much clearer (also, it will print out `'listErrorInfo null` in case of error which I think unintentional).